### ISSUE TITLE
Failure to generate mock of protocol omitting parameter name for an argument

### ISSUE THREAD

**Issue by pivotal-bolt** [Brightify/Cuckoo#288]
If a protocol omits parameter names for argument without labels, Cuckoo erroneously uses `name not set` in the generated mock which fails to compile. It should instead use `parameter1` or `nameNotSet` or some other valid Swift variable name.

Generating a mock of this protocol results in invalid Swift:
```
protocol AudioSessionManager {
    func allowBackgroundMusic(_: Bool)
}
```

In `GeneratedMocks.swift`, this becomes:
```
     func allowBackgroundMusic(_ name not set: Bool)  {
        
    return cuckoo_manager.call("allowBackgroundMusic(_: Bool)",
            parameters: (name not set),
            escapingParameters: (name not set),
            superclassCall:
                
                Cuckoo.MockManager.crashOnProtocolSuperclassCall()
                ,
            defaultCall: __defaultImplStub!.allowBackgroundMusic(name not set))
        
    }
```

**Comment by MatyasKriz** [comment]
Hey, nice issue you have there. It would be a shame if someone patched it.

Jokes aside, I didn't know you could do that in Swift. Nice catch!

Thanks to your report, I also found a bug concerning generic protocols that no one has reported yet, so thanks twice!

Fix for this issue has been released in `1.0.5`.

Please test it and if it works on your machine, close the issue.

**Comment by pivotal-bolt** [comment]
Just tested it out and it's working great, thank you!
